src/libraries/System.Net.Security/tests/FunctionalTests/TestHelper.cs

```diff
@@ -260,5 +261,15 @@ static string ProtocolToString(SslProtocols? protocol)
 
             return name;
         }
+
+        [ModuleInitializer]
+        internal static void InitializeQuic()
```

Comment:
We should move this into a file under Common and just include that file into the tests that need this workaround.